src/main/java/redis/clients/jedis/csc/CacheStats.java

```diff
@@ -0,0 +1,89 @@
+package redis.clients.jedis.csc;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+public class CacheStats {
```

Comment:
I like this. Will help us track some stats and see how the app works.

In the long run we should probably use a framework such as https://micrometer.io/ (Lettuce uses that)

Reply:
i ll give it a thought,,  a bit concerned about not caching when it is the expectation. 
imho warnings are tend to be out of sight when there are no bad things under investigation.

Reply:
lets keep that in mind for next phase of csc